worker/deps/libwebrtc/libwebrtc/modules/bitrate_controller/loss_based_bwe_v2.h

```diff
@@ -0,0 +1,183 @@
+    int num_lost_packets = 0;
+    int num_received_packets = 0;
+    DataRate sending_rate = DataRate::MinusInfinity();
+    int id = -1;
+  };
+
+  struct PartialObservation {
+    int num_packets = 0;
+    int num_lost_packets = 0;
+    DataSize size = DataSize::Zero();
+  };
+
+  static absl::optional<Config> CreateConfig(
+      const WebRtcKeyValueConfig* key_value_config);
+  bool IsConfigValid() const;
+
+  // Returns `0.0` if not enough loss statistics have been received.
+  double GetAverageReportedLossRatio() const;
+  std::vector<ChannelParameters> GetCandidates(
+      DataRate delay_based_estimate) const;
+  DataRate GetCandidateBandwidthUpperBound(DataRate delay_based_estimate) const;
+  Derivatives GetDerivatives(const ChannelParameters& channel_parameters) const;
+  double GetFeasibleInherentLoss(
+      const ChannelParameters& channel_parameters) const;
+  double GetInherentLossUpperBound(DataRate bandwidth) const;
+  double AdjustBiasFactor(double loss_rate, double bias_factor) const;
+  double GetHighBandwidthBias(DataRate bandwidth) const;
+  double GetObjective(const ChannelParameters& channel_parameters) const;
+  DataRate GetSendingRate(DataRate instantaneous_sending_rate) const;
+  DataRate GetInstantUpperBound() const;
+  void CalculateInstantUpperBound();
+
+  void CalculateTemporalWeights();
+  void NewtonsMethodUpdate(ChannelParameters& channel_parameters) const;
+
+  // Returns false if there exists a kBwOverusing or kBwUnderusing in the
+  // window.
+  bool TrendlineEsimateAllowBitrateIncrease() const;
+
+  // Returns true if there exists an overusing state in the window.
+  bool TrendlineEsimateAllowEmergencyBackoff() const;
+
+  // Returns false if no observation was created.
+  bool PushBackObservation(std::vector<PacketResult> packet_results,
+                           BandwidthUsage delay_detector_state);
+  void UpdateTrendlineEstimator(
+      const std::vector<PacketResult> packet_feedbacks,
+      Timestamp at_time);
+  void UpdateDelayDetector(BandwidthUsage delay_detector_state);
+
+	void Reset();
+
+  absl::optional<DataRate> acknowledged_bitrate_;
+  absl::optional<Config> config_;
+  ChannelParameters current_estimate_;
+  int num_observations_ = 0;
+  std::vector<Observation> observations_;
+  PartialObservation partial_observation_;
+  Timestamp last_send_time_most_recent_observation_ = Timestamp::PlusInfinity();
+  Timestamp last_time_estimate_reduced_ = Timestamp::MinusInfinity();
+  absl::optional<DataRate> cached_instant_upper_bound_;
+  std::vector<double> instant_upper_bound_temporal_weights_;
+  std::vector<double> temporal_weights_;
+  std::deque<BandwidthUsage> delay_detector_states_;
+  Timestamp recovering_after_loss_timestamp_ = Timestamp::MinusInfinity();
+  DataRate bandwidth_limit_in_current_window_ = DataRate::PlusInfinity();
+  bool limited_due_to_loss_candidate_ = false;
+	// MS_NOTE changed min bitrate from 1 to 100, this allows faster recover
+	// from huge drop.
```

Comment:
Use spaces to indent in libwebrtc files.

```suggestion
  // NOTE: changed min bitrate from 1 to 100, this allows faster recover
  // from huge drop.
```